src/style.css

```diff
@@ -1 +1,63 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  margin: 0;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: Avenir;
+}
```

Comment:
You are setting the font to Avenir, but you don't have a fallback font. Remember to use fallback fonts - alternative font-family in case the main one doesn't work.